<ui version="4.0" >
 <class>TrackerView</class>
 <widget class="QWidget" name="TrackerView" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1049</width>
    <height>246</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>URL:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="m_url" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>Status:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="m_status" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QLabel" name="label_5" >
         <property name="text" >
          <string>Next Update In:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="m_next_update" >
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="1" >
    <widget class="KPushButton" name="m_update_tracker" >
     <property name="text" >
      <string>Update Tracker</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLineEdit" name="m_tracker_to_add" />
   </item>
   <item row="1" column="1" >
    <widget class="KPushButton" name="m_add_tracker" >
     <property name="text" >
      <string>Add Tracker</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QListWidget" name="m_tracker_list" />
   </item>
   <item row="2" column="1" >
    <layout class="QVBoxLayout" >
     <item>
      <widget class="KPushButton" name="m_remove_tracker" >
       <property name="text" >
        <string>Remove Tracker</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="m_change_tracker" >
       <property name="text" >
        <string>Change Tracker</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>81</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="KPushButton" name="m_restore_defaults" >
       <property name="text" >
        <string>Restore Defaults</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header location="global" >kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
